app/view_models/auction_show_view_model.rb

```diff
@@ -73,7 +73,9 @@ def bid_status_class
   end
 
   def bid_status_partial
-    if auction.type == 'reverse' && (over? || available_and_user_is_winning_bidder?)
+    if reverse_auction_over? ||
```

Comment:
interesting! if they are failing silently I wonder if there are tests or changes we can make that will make these more obvious...

One certain thing we could do would be to keep those types on a constant so that they are more greppable and raise when they are changed or do not exist.